fcrepo-indexing-triplestore/src/main/java/org/fcrepo/camel/indexing/triplestore/FcrepoTripleStoreIndexerConfig.java

```diff
@@ -70,6 +70,12 @@
     @Value("${triplestore.baseUrl:http://localhost:8080/fuseki/test/update}")
     private String triplestoreBaseUrl;
 
+    @Value("${triplestore.authUsername:}")
```

Comment:
Can you drop these two properties in the root README.md file?

Reply:
done!